generic/tests/cfg/unattended_install.cfg

```diff
@@ -83,8 +83,22 @@
         - large_image:
             only unattended_install.cdrom
             only qcow2 qcow2v3
-            image_size_image1 = 2300G
-            image_size = 2300G
+            install_timeout = 7200
+            RHEL.6:
```

Comment:
Ok, others are good to me.
But please update he commit log:
s/unattended_install, large_image/unattended_install.large_image